docs/releasehistory.rst

```diff
@@ -1,12 +1,23 @@
 Release History
 ***************
 
-0.21.2 - Bugfix release
+0.21.3 - Bugfix release
 =======================
 
 Bugfixes
 --------
-- Fixed bug with samplers not broadcasting mixed replicas to MPI context (`#449 <https://github.com/choderalab/openmmtools/pull/449>`_).
+- Fixed long-standing bug with replica mixing using multiple GPUs  (`#449 <https://github.com/choderalab/openmmtools/pull/449>`_) & (`#562  <https://github.com/choderalab/openmmtools/pull/562>`_).
```

Comment:
Can we mention what release we think we introduced this into? Or was it always present with the MPI code?
We should explicitly mention MPI as well.

Reply:
Can we mention which issue this fixes?

Reply:
e.g. "Bug in replica mixing in MPI multi-GPU runs--where some replicas were simulated in incorrect states--was fixed. Believed to be present since Release X.Y".

Reply:
Yes, thanks for the suggestion. It was always present, because it was ported from Yank.